README.md

```diff
@@ -63,4 +63,4 @@ Every help on this project is greatly appreciated. To get you started, here's a
 
 [MIT](https://github.com/philipp-spiess/create-store/blob/master/README.md)
 
-[Redux]: https://redux.js.org/introduction
+[Redux](https://redux.js.org/introduction)
```

Comment:
This is part of the [reference-style link](https://daringfireball.net/projects/markdown/syntax#link) from above.

```suggestion
[Redux]: https://redux.js.org/introduction
```